src/devices/namron.ts

```diff
@@ -1624,4 +1624,43 @@ export const definitions: DefinitionWithExtend[] = [
             m.temperature({reporting: undefined}),
         ],
     },
+    {
+        zigbeeModel: ["4512791"],
+        model: "4512791",
+        vendor: "Namron AS",
+        description: "Namron Simplify Zigbee dimmer (1/2-polet / Zigbee / BT) ",
+        fromZigbee: [fz.on_off, fz.brightness, fz.power_on_behavior],
+        toZigbee: [tz.on_off, tz.light_onoff_brightness, tz.power_on_behavior],
```

Comment:
please use modern extend instead of to/fromZigbee (search for m.light and m.electricityMeter in this codebase for examples